gsa/src/web/pages/cves/details.js

```diff
@@ -69,44 +69,41 @@ const CveDetails = ({entity}) => {
             <TableRow>
               <TableData>{_('Base Score')}</TableData>
               <TableData>
-                <SeverityBar severity={severity} />
+                <SeverityBar severity={entity.score / 10} />
```

Comment:
We should probably parse this in parseObject in the model like it was done by parseElement before.